position:
    - home_pose: [3.25, 9.61, -1.6]
    - kitchen_pose: [1.65, 8.71, 1.57]
    - table_1 pose: [ 1.53,  -1.78, -1.57]
    - table_2 pose: [ 1.34,  -5.74, -1.57]
    - table_3 pose: [ -0.55,  -5.74, -1.57]
